.drone/drone.jsonnet

```diff
@@ -36,58 +36,70 @@ local docker_username_secret = secret('docker_username', 'infra/data/ci/docker_h
 local docker_password_secret = secret('docker_password', 'infra/data/ci/docker_hub', 'password');
 
 // secrets for pushing serverless code packages
-local fn_upload_ops_tools_secret = secret('ops_tools_fn_upload', 'infra/data/ci/tempo-ops-tools-function-upload', 'credentials.json');  
+local fn_upload_ops_tools_secret = secret('ops_tools_fn_upload', 'infra/data/ci/tempo-ops-tools-function-upload', 'credentials.json');
 
 // secret needed to access us.gcr.io in deploy_to_dev()
 local docker_config_json_secret = secret('dockerconfigjson', 'secret/data/common/gcr', '.dockerconfigjson');
 
 // secret needed for dep-tools
 local gh_token_secret = secret('gh_token', 'infra/data/ci/github/grafanabot', 'pat');
 
-# gcs buckets to copy serverless functions to
-local gcp_secrets = [ fn_upload_ops_tools_secret.name ];
-local gcp_serverless_deployments = [ 
-  { 
+// gcs buckets to copy serverless functions to
+local gcp_secrets = [fn_upload_ops_tools_secret.name];
+local gcp_serverless_deployments = [
+  {
     bucket: 'ops-tools-tempo-function-source',
     secret: fn_upload_ops_tools_secret.name,
   },
-  { 
+  {
     bucket: 'grafanalabs-global-tempo-function-source',
     secret: fn_upload_ops_tools_secret.name,
   },
 ];
 
-## Steps ##
+local aws_serverless_deployments = [
+  {
+    local cluster = 'dev-us-east-0',
+    local namespace = 'tempo-dev-02',
+    bucket: '%s-%s-fn-source' % [cluster, namespace],
+    secrets: [
```

Comment:
i don't see why this is a slice. should this not be an object of some kind:

```
{
  access_key: <>,
  secret_key: <>
}
```

Technically this could work as a slice I guess, but these are known/named fields so an object would probably be a clearer way to implement this.